nft_ingester/src/main.rs

```diff
@@ -1,59 +1,46 @@
-mod bubblegum_indexer;
-mod crud_indexer;
-mod gummyroll_indexer;
-mod utils;
+use std::sync::Arc;
+use futures_util::TryFutureExt;
+use solana_sdk::pubkey::Pubkey;
 
 use {
-    crate::{
-        bubblegum_indexer::handle_bubblegum_instruction,
-        crud_indexer::handle_gummyroll_crud_instruction,
-        gummyroll_indexer::handle_gummyroll_instruction,
-        utils::{filter_events_from_logs, order_instructions},
-    },
-    flatbuffers::{ForwardsUOffset, Vector},
-    lazy_static::lazy_static,
-    messenger::{ACCOUNT_STREAM, BLOCK_STREAM, SLOT_STREAM, TRANSACTION_STREAM},
+    messenger::{TRANSACTION_STREAM},
     plerkle::async_redis_messenger::AsyncRedisMessenger,
     plerkle_serialization::transaction_info_generated::transaction_info::root_as_transaction_info,
-    regex::Regex,
-    solana_sdk::pubkey::Pubkey,
     sqlx::{self, postgres::PgPoolOptions, Pool, Postgres},
 };
+use messenger::ACCOUNT_STREAM;
+use nft_ingester::parsers::{BubblegumHandler, GummyRollHandler, InstructionBundle, ProgramHandler, ProgramHandlerManager};
+use nft_ingester::utils::{order_instructions, parse_logs};
+use plerkle_serialization::account_info_generated::account_info::root_as_account_info;
```

Comment:
would like to stick just one type of import, I'm fine with either `use {}` or  a lot of smaller `use plerkle::` etc.